app/src/main/res/layout/fragment_create_emoji_id.xml

```diff
@@ -0,0 +1,237 @@
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/txt_just_sec_title"
+                android:layout_gravity="center"
+                android:gravity="center"
+                android:text="@string/create_emoji_id_just_sec_desc_text"
+                android:textColor="@color/black"
+                android:textSize="18sp"
+                android:visibility="invisible"
+                app:customFont="AVENIR_LT_STD_ROMAN" />
+
+            <View
+                android:id="@+id/just_sec_back_view"
+                android:layout_width="match_parent"
+                android:layout_height="25dp"
+                android:layout_below="@id/txt_just_sec_title"
+                android:layout_gravity="center"
+                android:background="@color/create_emoji_id_bg"
+                android:visibility="invisible" />
+
+            <com.tari.android.wallet.ui.component.CustomFontTextView
+                android:id="@+id/txt_wallet_address_des"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/txt_create_your_emoji_id"
+                android:layout_gravity="center"
+                android:alpha="0"
+                android:gravity="center"
+                android:lineSpacingExtra="4dp"
+                android:text="@string/create_emoji_id_wallet_address_desc"
+                android:textColor="@color/dark_gray"
+                android:textSize="14sp"
+                android:visibility="visible"
+                app:customFont="AVENIR_LT_STD_MEDIUM" />
+
+            <com.airbnb.lottie.LottieAnimationView
+                android:id="@+id/nerd_face_emoji"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_centerHorizontal="true"
+                android:layout_marginTop="10dp"
+                android:alpha="0"
+                app:lottie_autoPlay="false"
+                app:lottie_enableMergePathsForKitKatAndAbove="true"
+                app:lottie_imageAssetsFolder="images/"
+                app:lottie_loop="false"
+                app:lottie_rawRes="@raw/nerd_emoji" />
+
+            <com.tari.android.wallet.ui.component.CustomFontTextView
+                android:id="@+id/txt_awesome"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="25dp"
+                android:gravity="center"
+                android:text="@string/create_emoji_id_awesome"
+                android:textColor="@color/black"
+                android:textSize="18sp"
+                android:visibility="invisible"
+                app:customFont="AVENIR_LT_STD_BLACK" />
+
+            <View
+                android:id="@+id/awesome_text_back_view"
```

Comment:
@jigneshsanghani When you're id'ing your components can you please add the name of the context (activity/fragment) as a prefix? Afaik this is the only way to apply namespaces/packages to id's. Please have a look at one of the other layout files for examples. This should apply to all your UI components.